x-pack/platform/plugins/shared/logs_shared/public/components/logging/log_text_stream/text_styles.tsx

```diff
@@ -6,35 +6,39 @@
  */
 
 import React, { useMemo, useState, useCallback } from 'react';
-
-import { euiStyled, css } from '@kbn/kibana-react-plugin/common';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { EuiThemeComputed, useEuiFontSize } from '@elastic/eui';
+import { withAttrs } from '../../../utils/theme_utils/with_attrs';
 import { TextScale } from '../../../../common/log_text_scale';
 
 export type WrapMode = 'none' | 'pre-wrapped' | 'long';
 
-export const monospaceTextStyle = (scale: TextScale) => css`
-  font-family: ${(props) => props.theme.eui.euiCodeFontFamily};
-  font-size: ${(props) => {
-    switch (scale) {
-      case 'large':
-        return props.theme.eui.euiFontSizeM;
-      case 'medium':
-        return props.theme.eui.euiFontSizeS;
-      case 'small':
-        return props.theme.eui.euiFontSizeXS;
-      default:
-        return props.theme.eui.euiFontSize;
-    }
-  }};
-  line-height: ${(props) => props.theme.eui.euiLineHeight};
+const getFontSize = (textscale: TextScale) => {
+  switch (textscale) {
+    case 'large':
+      return 'm';
+    case 'medium':
+      return 's';
+    case 'small':
+      return 'xs';
+    default:
+      return 's';
+  }
+};
+
+export const useMonospaceTextStyle = (scale: TextScale, theme: EuiThemeComputed) => css`
```

Comment:
Nit: Should we call the param `euiTheme` like the other places (hoveredContentStyle, highlightedContentStyle) or change it there to `theme` or there is a difference in the theme used (I see we use the same type)? just a nit it's up to you how you want to name them :)

Reply:
good catch